File: ratings/src/main/java/com/cinelist/ms/ratings/dtos/client/MovieResponse.java
package com.cinelist.ms.ratings.dtos.client;

import java.time.LocalDate;
import java.util.UUID;

public record MovieResponse(
        UUID identifier,
        String title,
        String shortDescription,
        String longDescription,
        LocalDate releaseDate,
        String trailerUrl,
        String thumbnailUrl,
        UUID certificateIdentifier,
        UUID languageIdentifier,
        String duration
) {
}

File: accounts/src/main/kotlin/com/cinelits/ms/accounts/dtos/client/MovieResponse.kt
package com.cinelits.ms.accounts.dtos.client

import java.time.LocalDate
import java.util.UUID

data class MovieResponse(
    val identifier: UUID,
    val title: String?,
    val shortDescription: String?,
    val releaseDate: LocalDate?,
    val trailerUrl: String?,
    val thumbnailUrl: String?,
    val certificateIdentifier: UUID?,
    val languageIdentifier: UUID?,
    val duration: String?
)